### ISSUE TITLE
NTR: CT, MRI or angiography: muscle AVM (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8944]
New term request CT, MRI or angiography: muscle AVM (elthat_110121014539): Arteriovenous malformation

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000586
- label: Muscle arteriovenous malformation
- parents: [Arteriovenous malformation (HP:0100026)](https://hpo.jax.org/app/browse/term/HP:0100026)
- pmid: 
- def: An anomalous configuration of blood vessels that shunts arterial blood directly into veins without passing through the capillaries and that is located in muscle tissue.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8944
- synonyms: Muscle AVM(NONE)|Arteriovenous malformation in muscle(NONE)
